NUModules/NUModuleAssignation.j

```diff
@@ -233,10 +233,26 @@ NUModuleAssignationActionUnassign = @"NUModuleAssignationctionUnassign";
 #pragma mark -
 #pragma mark Assignation & Unassignation
 
-/*! Opens the assign object chooser
+/*! Opens the assign object confirmation window
 */
 - (@action)openAssignObjectPopover:(id)aSender
```

Comment:
I would say let's keep the name as openAssignObjectPopover for choosing the objects to assign and rename the new function openAssignConfirmation or something of this sort to be more appropriate with the functionality